### PR TITLE
Add Mint L2

### PR DESCRIPTION
**Pull request by zhwindy**
This PR Is requesting to update the information of Mint Blockchain on L2Beat.

Mint Blockchain is a Layer 2 network for NFTS.
Homepage: https://www.mintchain.io/
Twitter: https://twitter.com/Mint_Blockchain
Testnet will be launched soon.

### CODE REVIEW

**Comment by enamul1590** [comment]
https://twitter.com/Enamul1590

**Comment by 0x1992** [comment]
Hi @lucadonnoh , can you reveiw this request? 

Mint is the L2 Blockchain for NFTs. Mint blockchain is developed based on OP Stack. Mint Testnet will be launched in Q1 2024.
Website: https://www.mintchain.io/
Twitter: https://twitter.com/Mint_Blockchain

Thanks for your time!

**Comment by mateuszradomski** [comment]
Hi, please format the `mint.ts` file. You can do that by running `yarn format:fix` in the `config` subpackage

**Comment by zhwindy** [comment]
> Hi, please format the `mint.ts` file. You can do that by running `yarn format:fix` in the `config` subpackage

ok, the `mint.ts` file is format already.

File: packages/frontend/src/static/icons/mint.png
Comment (lucadonnoh) [inline packages/frontend/src/static/icons/mint.png:1]: icon should be a 256x256 png

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

File: packages/frontend/src/static/icons/mint.png
Comment (zhwindy) [reply packages/frontend/src/static/icons/mint.png:1]: ok, the icon is changed to 256x256 png already

**Comment by lucadonnoh** [comment]
do you have a live testnet already?

**Comment by zhwindy** [comment]
> do you have a live testnet already?

we now have an internal testnet, the public testnet is not yet online

**Comment by zhwindy** [comment]
@lucadonnoh hi, is there any other information that needs to be modified or submitted?

**Review by lucadonnoh — CHANGES_REQUESTED** [review]

**Review by lucadonnoh — APPROVED** [review]

**Pull request merged**